cfg/TebLocalPlannerReconfigure.cfg

```
@@ -141,6 +141,10 @@ grp_robot_omni = grp_robot.add_group("Omnidirectional", type="hide")
 
 grp_robot_omni.add("max_vel_y", double_t, 0, 
   "Maximum strafing velocity of the robot (should be zero for non-holonomic robots!)",
+  0.0, 0.0, 100)
+
+grp_robot_omni.add("max_vel_linear", double_t, 0, 
```

```suggestion
grp_robot_omni.add("max_vel_trans", double_t, 0, 
```
S: rename from _linear_ to _translational_.

---

linear velocity is vastly more common in the literature. Is there a specific reason you want to change it to translational?

---

Translational seems to be the common name to use in ROS planners (e.g. [dwa](http://wiki.ros.org/dwa_local_planner) and [eband](http://wiki.ros.org/eband_local_planner))

---

that's the name on DWA,,, and also we have below `trans_stopped_vel`